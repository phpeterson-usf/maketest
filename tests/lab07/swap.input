-h $(repo_dir)/swap.hex $(TESTS_DIR)/swap_s.s
