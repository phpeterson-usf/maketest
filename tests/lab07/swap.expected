v2.0 raw
E5902000
E5913000
E5812000
E5803000
E12FFF1E
